	/**
	 * Throwing exception confirms that value is received.
	 */
	public void testSubmit_ValidTextField_NoFile()
	{
		formTester.setValue(textFieldId, FileUploadError.THIS_VALUE_SHOULD_THROW_EXCEPTION);
		try
		{
			formTester.submit();
			fail("Value not succesfully submitted.");
		}
		catch (WicketRuntimeException rex)
		{
			Throwable ex = rex.getCause().getCause();
			assertEquals("Special value: " + FileUploadError.THIS_VALUE_SHOULD_THROW_EXCEPTION,
				ex.getMessage());
		}
	}

